letters/modernized/02-letters.adoc

```diff
@@ -1,7 +1,7 @@
-== Letters I &mdash; XVI
+== Letters &mdash; XVI
```

Comment:
I couldn't remember what I erased here when I did it, and then I wasn't even sure how you wanted this to look so I just left it and figured you could tell me here. :)